### ISSUE TITLE
Votive.VSIX install fails in VS2017Community on Windows 10

### ISSUE THREAD

**Issue by ewhitchu** [wixtoolset/issues#5633]
[VSIXInstaller_d196912e-01aa-4d4c-81c5-f4c0b42c37fe.zip](https://github.com/wixtoolset/issues/files/1124574/VSIXInstaller_d196912e-01aa-4d4c-81c5-f4c0b42c37fe.zip)

Please provide answers to the following questions to help us narrow down, reproduce, and fix the problem. Fill out one section and delete the others.

## Triage

Issues are triaged at [online meetings](https://www.firegiant.com/blog/), generally held alternate Tuesdays at 9:30 a.m. Pacific time (UTC-7/UTC-8). Meeting requests are sent to the [wix-devs](http://wixtoolset.org/documentation/mailinglist/#wix-devs) mailing list.

## Discussions and support

Note that discussions are best held on the mailing lists, not in issues. Please do not open issues requesting support or debugging help; first start a thread on the [wix-users](http://wixtoolset.org/documentation/mailinglist/#wix-users) mailing list and open an issue if the discussion suggests a bug in the WiX Toolset or that a feature request is appropriate.

## Bugs

If this issue is a bug:

* Which version of WiX are you building with?

> 3.11stable downloaded several times in last month

* Which version of Visual Studio are you building with (if any)?

> Visual Studio Community 2017 v15.2 (26430.14) Release if Votive worked!
* Which version of the WiX Toolset Visual Studio Extension are you building with (if any)?

> Votive 3.11 downloaded today 5th Jul 2017)

* Which version of .NET are you building with?

> Various
* If the problem occurs when installing your packages built with WiX, what is the version of Windows the package is running on?

> Irrelevant

* Describe the problem and the steps to reproduce it.

> Pre-installed Wix3.11 exe. When prompted, started the Votive install. VSIX did not complete successfully. Log showed errors. 
Note that I have SQL Server 2014 Dev, and SQL Server 2016 Express installed

* Describe the behavior you expected and how it differed from the actual behavior.

> Log attached Had no issues with VS Community 2015

**Comment by barnson** [comment]
You have files from WiX hacked into the VS directory tree. You need to remove those files before you can install the VSIX.

**Comment by ewhitchu** [comment]
Thanks, I will see what I can find. I have done nothing other than try the install a few times. You wouldn't know which files would you?

**Comment by robmen** [comment]
Log file says:

> 5/07/2017 10:13:03 PM - Package 'WixToolset.VisualStudioExtension.Dev15,version=0.9.21.62588' failed to install. System.IO.IOException: The file 'C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\MSBuild\Microsoft\WiX\v3.x\wix.ca.targets' already exists.

**Comment by ewhitchu** [comment]
Thanks.

It all went bad when I first tried to update from 3.10 to 3.11. Anyway, I have uninstalled VS2017 and cleared out anything in its Path – it does not clean up after itself!.

I had installed two Wix-related VSIX extensions – Wax, and Xeam Visual Installer. The latter, as with Votive, apparently has no uninstall capability. (It has however, saved me a lot in designing for Burn.

Reinstalling and starting from scratch. Messy but necessary I think. I just hope I do not have issues re-establishing connections to my Git repos in Visual Studio Team in Azure.

Thanks, I have been using Wix for several years and am more than happy with it. This is the first major issue and it still looks like a bug with the Votive VSIX to me.

Eric Whitchurch

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10
